src/EditorFeatures/Core.Wpf/WpfClassificationExtensions.cs

```diff
@@ -66,15 +66,22 @@ public static Run ToRun(this ClassifiedText part, IClassificationFormatMap forma
             ClassificationTypeMap typeMap)
         {
             var inlines = parts.ToInlines(formatMap, typeMap);
-            return inlines.ToTextBlock(formatMap, typeMap);
+            return inlines.ToTextBlock(formatMap);
         }
 
+        [Obsolete("Use the method redirected")]
```

Comment:
Give the fully qualified name of where this is redirecting to. Otherwise this is going to result in head scratching for whoever sees this.

Reply:
Or maybe "Use the overload that doesn't take the typeMap" so it's a bit clearer.

Reply:
I think the first suggestion is more transparent. Thank you!

---
In reply to: [247292817](https://github.com/dotnet/roslyn/pull/32307#discussion_r247292817) [](ancestors = 247292817)